MeetU/MeetU/Scripts/Controllers/Meetup/index.js

```diff
@@ -2,207 +2,213 @@
     "use strict";
     angular
         .module('meetupModule', ['ngResource', 'angularMoment', 'ngRoute', 'angular-confirm', 'ui.bootstrap.tpls'])
-        .controller('meetupIndexController', function ($scope, $resource, $q, $log, $confirm) {
+        .controller('meetupIndexController',meetupIndexController)
+         meetupIndexController.$inject=["$scope", "$resource", "$q", "$log", "$confirm"]      
+         function meetupIndexController($scope, $resource, $q, $log, $confirm) {
 
-            var currentShowingMeetupCount = 0;
-            var MEETUPS_PER_PAGE = 5;
-            //
-            //  Lazy resources
-            //
-            var Meetup = $resource('/api/Meetups');
-            var Userview = $resource('/api/loggedUser');
-            var Join = $resource('/api/Joins');
-            var CommentView = $resource('/api/Comments/');
-            //
-            //  Queries
-            //
+        var currentShowingMeetupCount = 0;
```

Comment:
it seems the indention isn't quite right? which causes lots of code changes here